scheds/kernel-examples/scx_flatcg.bpf.c

```diff
@@ -284,14 +285,16 @@ static void cgrp_enqueued(struct cgroup *cgrp, struct fcg_cgrp_ctx *cgc)
 	}
 
 	stash = bpf_map_lookup_elem(&cgv_node_stash, &cgid);
-	if (!stash) {
+	if (!stash || !stash->node) {
 		scx_bpf_error("cgv_node lookup failed for cgid %llu", cgid);
 		return;
 	}
 
-	/* NULL if the node is already on the rbtree */
-	cgv_node = bpf_kptr_xchg(&stash->node, NULL);
+	cgv_node = bpf_refcount_acquire(stash->node);
 	if (!cgv_node) {
+		/* Node never leaves cgv_node_stash, this should only happen if
```

Comment:
Minor nit, but can you bring this comment to a separate line? I don't _think_ we have any multi-line comments that aren't of the form
```
/*
 * stuff
 */ 
```